docs/source/installation.rst

```diff
@@ -0,0 +1,31 @@
+.. _installation:
+
+Installation
+============
+
+For prerequisites and optional dependencies, see :doc:`dependencies`.
+
+Install or upgrade with pip
+---------------------------
+
+Use this command for both first-time installs and upgrades:
+
+.. code-block:: bash
+
+    python -m pip install --upgrade urlwatch
+
+Verify the installation
+-----------------------
+
+Check that urlwatch is on your ``PATH`` and prints a version number:
+
+.. code-block:: bash
+
+    urlwatch --version
+
+Next steps
+----------
+
+Continue with :doc:`introduction` for the Quick Start and initial
+configuration. For optional dependencies that enable additional filters and
```

Comment:
The "For optional dependencies that enable additional filters and reporters, see :doc:`dependencies`." part is already written above:

> For prerequisites and optional dependencies, see :doc:`dependencies`.